yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1548,6 +1548,7 @@ func addAssessmentIssuesForUnsupportedDatatypes(unsupportedDatatypes []utils.Tab
 				log.Warnf("failed to get datatype from %s", colInfo.DataType)
 				continue
 			}
+			datatype = strings.TrimSuffix(datatype, "[]") // for the array types we add the '[]' to the type for distinguish between normal type and array based datatype
```

Comment:
Let's handle this for other scenarios as well. (write a common function as discussed)